emails/shared/base-email.pug

```diff
@@ -1,40 +1,40 @@
 doctype html
 // all styles/assets need to be relative to the "emails" directory AKA /emails
-link(rel="stylesheet", href="/shared/style.css", data-inline)
+link(rel="stylesheet", href="./style.css", data-inline)
```

Comment:
Due to the way "extending" a template works, all styles/assets need to be relative to the emails directory. Keep the `/shared/style.css` (see above comment)